identity_derive/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
+name = "identity_derive"
```

Comment:
Might be better to name this `identity_diff_derive`, or is this meant to contain a variety of `derive`able behaviours?

Reply:
It may have other derivation macros if we decide to add any.  At least that was the intent.